### ISSUE TITLE
2.0.x versions not sending commands on Mac

### ISSUE THREAD

**Issue by psilentrain1** [JeffreyDavidsz/VICREO-Listener#35]
I'm on MacOS 10.14 Mojave running Companion v1.4.0
After setting up some key combinations and hotkeys in companion, I see input in VICREO's log, but the key presses aren't making it to the window that's in focus. 

I'm getting this same behavior in 2.0.7, 2.0.6, and 2.0.5. The commands work as expected in 1.4.0.

Is there anything I'm missing?

**Comment by psilentrain1** [comment]
Recent logs from 2.0.7:

0:13:11 please change to new syntax
0:13:11 <FILE>/Applications/zoom.us.app
0:13:10 please change to new syntax
0:13:10 <FILE>/Applications/zoom.us.app
0:13:9 please change to new syntax
0:13:9 <FILE>/Applications/zoom.us.app
0:13:7 <KCOMBO>cmd<AND>tab
0:13:7 <KCOMBO>cmd<AND>tab
0:13:6 <KCOMBO>cmd<AND>tab
0:12:51 connected

**Comment by psilentrain1** [comment]
Update: Upgraded to Companion v2.0.0 and Listener 2.0.7 works as expected!

**Comment by JeffreyDavidsz** [comment]
Hi glad that this worked out. Would you be able to point me out what you tried as keys? There is a small difference but it should also work on 1.4 of Companion

**Comment by psilentrain1** [comment]
@JeffreyDavidsz Thanks for your response!

I was working on a way to remotely control Zoom, so I was using the following combos:
Command+J
Command+Shift+A
Command+Shift+V
Command+Shift+W
Control+N
Control+P
Option+Y

For testing outside of Zoom, I also configured buttons for Command+tab, Arrow Up, and Arrow Down.

None of these buttons/combos worked in Companion 1.4 unless I used listener 1.4.

Thanks again!
